app/src/main/java/com/example/fooding/fragments/PantryFragment.java

```diff
@@ -31,6 +40,8 @@ public class PantryFragment extends Fragment implements SearchView.OnQueryTextLi
     private SearchView recipeSearchView;
     private RecyclerView ingredientsRecyclerView;
     private RecyclerView suggestedRecyclerView;
+    private TextView ingredientName;
+    private TextView ingredientImage;
```

Comment:
A small nit pick. `ingredientName` and `ingredientImage` are `TextView`'s, so perhaps rename them to indicate they are `TextView` objects. 

```suggestion
    private TextView ingredientNameTextView;
    private TextView ingredientImageTextView;
```

Reply:
I have applied the changes. Thank you.